### ISSUE TITLE
Fails on LogAnalytics and ObjectStore when executed outside the home region.

### ISSUE THREAD

**Issue by danielwoodz** [therealcmj/ociextirpater#2]
If the script is run in the home region solely, there is no issue. If it is run in the home region + any other region, there is no issue. If it is run only in a non-home region, on certain resources you will get:
```python
Traceback (most recent call last):
  File "/Users/danielwoodz/code/ociextirpater/ociextirpate.py", line 35, in <module>
    o = cls( cfg )
        ^^^^^^^^^^
  File "/Users/danielwoodz/code/ociextirpater/ociextirpater/ociclients/loganalytics.py", line 16, in __init__
    self.namespace = self.clients[ config.home_region ].list_namespaces(config.ociconfig["tenancy"]).data.items[0].namespace_name
                     ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'us-phoenix-1'

Traceback (most recent call last):
  File "/Users/danielwoodz/code/ociextirpater/ociextirpate.py", line 35, in <module>
    o = cls( cfg )
        ^^^^^^^^^^
  File "/Users/danielwoodz/code/ociextirpater/ociextirpater/ociclients/objectstore.py", line 71, in __init__
    self.namespace = self.clients[ config.home_region ].get_namespace().data
                     ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'us-phoenix-1'
```

I wrapped these locally in a try/except to exercise the whole script and find all of them. I assume this is something to do with default regions only resources, but am unsure if try/except and returning keeps full coverage.

**Comment by therealcmj** [comment]
Oh! Yeah that's not great. I suspect it affects more than those items. Let me get a fix in on that this week.

**Comment by therealcmj** [comment]
Fixed in commit 32e16ab. Thanks for letting me know!
